packages/@webex/webex-core/src/lib/services/services.js

```diff
@@ -941,6 +944,11 @@ const Services = WebexPlugin.extend({
         catalog.updateServiceUrls('override', formattedOverrideServices);
       }
 
+      // if not fedramp, append on the commercialAllowedDomains
+      if (!fedramp) {
+        services.allowedDomains = [...services.allowedDomains, ...COMMERCIAL_ALLOWED_DOMAINS];
```

Comment:
I see in the service-catalog file we have used the union to merge 2 arrays to ensure that allowedDomains list is unique. Would there be no possibility to have duplicates here between services.AllowedDomains and COMMERCIAL_ALLOWED_DOMAINS?

Reply:
valid point. will change